af/fastpath/fastpath/core.py

```diff
@@ -1429,6 +1424,25 @@ def score_riseupvpn(msm: dict) -> dict:
     return scores
 
 
+def score_openvpn(msm: dict) -> dict:
+    scores = init_scores()
+    tk = g_or(msm, "test_keys", {})
+    tstatus = tk.get("transport_status") or {}
```

Comment:
This scoring logic looks like it's scoring a `riseupvpn` test instead of an `openvpn` one based on the specs I see here:
* https://github.com/ooni/spec/blob/f9bbaa83541484e3e509ffa56dd87b0c5ce8c31a/nettests/ts-026-riseupvpn.md
* https://github.com/ooni/spec/pull/255/commits/2e8887f03fae18b50a8fa94b0ffc0c2f728af3c5

Reply:
indeed, this seems to be coming from riseupvpn

Reply:
It was "stub" code as part of the WIP code, now fixed.